src/CMakeLists.txt

```diff
@@ -41,8 +41,8 @@ if(CMAKE_C_COMPILER_ID MATCHES "^(Intel)$")
 elseif(CMAKE_C_COMPILER_ID MATCHES "^(GNU|Clang|AppleClang)$")
   set(CMAKE_C_FLAGS "-g ${CMAKE_C_FLAGS}")
   set(CMAKE_C_FLAGS_DEBUG "-ggdb -Wall")
-  set(CMAKE_Fortran_FLAGS "-g -fno-range-check -funroll-loops ${CMAKE_Fortran_FLAGS}")
-  set(CMAKE_Fortran_FLAGS_DEBUG "-ggdb -Wall")
+  set(CMAKE_Fortran_FLAGS "-g -fno-range-check -funroll-loops -fsanitize=address ${CMAKE_Fortran_FLAGS}")
+  set(CMAKE_Fortran_FLAGS_DEBUG "-ggdb -Wall -fsanitize=address")
```

Comment:
We don't set this in the CMake, but rather on the command line when running CMake. Use -DCMAKE_FORTRAN_FLAGS="-fsanitize=address" when running CMake.

Also you need to do it on the C flags at the same time, if C can be involved. (And it can be with g2.)

So use something like:

`cmake -DDCMAKE_FORTRAN_FLAGS="-g -Wall -fsanitize=address" -DDCMAKE_C_FLAGS="-g -Wall -fsanitize=address" ..`